MapboxNavigation/RouteController.swift

```diff
@@ -24,6 +24,12 @@ open class RouteController: NSObject {
     
     
     /*
+     If true, the user puck is snapped to closest location on the route.
+    */
+    public var snapUserToRoute = false
```

Comment:
“Snap user to route” sounds like a command, so this sounds like an action method instead of a property that can be set. (In Objective-C, an action method and a property getter call can look identical.) Also, it sounds funny to say that we’re moving the user themselves; we’re actually moving the user location annotation.

Let’s call this property `snapsUserLocationAnnotationToRoute`. It’s a mouthful, but the developer won’t have to call it often anyways.